test/jobs.test.js

```diff
@@ -41,23 +50,55 @@ describe('Jobs Unit Test', () => {
         mockery.disable();
     });
 
+    describe('redis constructor', () => {
+        it('creates a redis connection given a valid config', () => {
+            const redisConfig = {
+                database: 0,
+                pkg: 'ioredis',
+                host: '127.0.0.1',
+                options: {
+                    password: undefined
+                },
+                port: 6379
+            };
+
+            assert.calledWith(mockRedis, redisConfig);
+        });
+    });
+
     describe('start', () => {
         it('starts a job', (done) => {
-            const expectedConfig = { buildConfig: 'buildConfig' };
+            const expectedConfig = {
+                annotations: {
+                    'beta.screwdriver.cd/executor': 'screwdriver-executor-k8s'
```

Comment:
Though this is just a mock, but this should be 'beta.screwdriver.cd/executor': 'k8s'